docs/features/README_ja.md

```diff
@@ -62,8 +62,8 @@ Networking レイヤーが含まれています。
 
 ## Ostracon Features
 
-* [Extending Tendermint-BFT with VRF-based Election](consensus_ja.md)
-* [BLS Signature Aggregation](signature_aggregation_ja.md)
+* [Extending Tendermint-BFT with VRF-based Election](consensus/index_ja.md)
```

Comment:
Would you rename `index_ja.md` to `README.md`?

Reply:
The name "README" is conventionally intended to indicate "*the first page you should read*" that is located on the top or major category directory. (I think) Wouldn't it be better where the details page linked from there is not the "README"?

Reply:
After a bit of discussion, I'll rename to `README.md` with the intention that it'll appear as the default page on github, rather than "the first page you should read".